kolibri/core/assets/src/content-renderer.vue

```diff
@@ -14,25 +11,40 @@
 
   module.exports = {
     props: {
-      channelId: String,
-      contentId: String,
-      kind: String,
-      extension: String,
-      title: String,
       contentData: {
         type: Object,
-        default: () => ({ url: '#' }),
+        default: () => ({ pk: 0 }),
```

Comment:
I feel like any attributes in `contentData` that are special [1] should be explicitly passed as props. Yes this is a little redundant, but I think it would make the API of `content-renderer` more explicit.

In the code below, it looks like `content-renderer` makes special use of:
* `files`
* `kind`
* `extension`
* `pk`

[1] "special" meaning used by `content-renderer`, and not just passed through to the rendering plugin as "enumerables"